experiments/pages/sampleStyleGuidePage/README.md

```diff
@@ -0,0 +1,16 @@
+# Sample Style Guide Page
+
+Use a component-filled Skuid page to preview components and re-usable patterns to build a style guide for your Skuid project.
+
+As you're making changes to your Design System, you may need to see how well that change might sit with the rest of the design. So having a page that show all design patterns in your app can be useful. Also, you and other collaborators can adhere to this guide as they build more pages, ensuring a cohesive UI in the end.
+
+This page is built as a style guide for a sales application so you may see components or re-usable patterns that are relevant to this type of UI. However, for your project, you may have a different sets of patterns and components that work together to make up your Style Guide.
+
+## Usage
+Note: This example uses standard Salesforce objects for dummy data, so it should work in any Salesforce org.
+
+1. Import the `salesAppStyleGuide.designsystem` design system into your Salesforce org.
+2. Create a new Skuid page and paste/upload the xml file.
+3. Edit away.
+
+The design system included is not specific to Salesforce environment, so you can reuse the design system on a Skuid Platform site as well. You'd just need to build your own Style Guide page with your data instead of using the included xml.
```

Comment:
```suggestion
The design system included is not specific to Skuid on Salesforce, so you can reuse the design system on a Skuid Platform site as well. You'd just need to build your own style guide page with your data instead of using the included XML.
```